haicrypt/cryspr-openssl.c

```diff
@@ -192,8 +233,13 @@ CRYSPR_methods *crysprOpenSSL(void)
     #endif
 
     //--Crypto Session API-----------------------------------------
+#ifdef CRYSPR2
+        crysprOpenSSL_methods.open     = crysprOpenSSL_Open;
+        crysprOpenSSL_methods.close    = crysprOpenSSL_Close;
```

Comment:
If `crysprOpenSSL_Close` only redirects to `crysprHelper_Close` (that is, it's a kinda basic universal close, not to be overridden), then here it should be fine to use `crysprHelper_Close`. Just suggesting.

Reply:
Out of my mind, custom open called for custom close. But it is true that Allocated buffer size and structure in open is irrelevant in close. I'll do it in the name of simplicity, gone the custom close.

Reply:
Fixed as suggested.